README.md

```diff
@@ -5,35 +5,40 @@
 
 ## Get started
 
-1. Install KubeSphere via [kk](https://github.com/kubesphere/kubekey/) (or other ways). 
-    This is an optional step, basically we need a Kubernetes Cluster and the front-end of DevOps.
-1. Install `ks-devops` via [chart](charts/ks-devops)
-1. Replace the images of `ks-apiserver` and `ks-controller-manager`. In current phase, we need to use a temporary images of [KubeSphere](https://github.com/kubesphere/kubesphere/) 
-which comes from [the branch remove-devops-ctrl](https://github.com/LinuxSuRen/kubesphere/tree/remove-devops-ctrl):
+### Quick Start
 
-* `kubespheredev/ks-apiserver:remove-devops-ctrl`
-* `kubespheredev/ks-controller-manager:remove-devops-ctrl`
+- Install KubeSphere via [kk](https://github.com/kubesphere/kubekey/) (or other ways).
+  This is an optional step, basically we need a Kubernetes Cluster and the front-end of DevOps.
```

Comment:
This statement might make people feel confused. I prefer to remove it here.

For those users who want to try DevOps in Kubernetes, then you can put this statement there.